CHANGELOG.md

```diff
@@ -38,6 +38,8 @@
 * Add `BracesRequiredMethods` parameter to `Style/BlockDelimiters` to require braces for specific methods such as Sorbet's `sig`. ([@maxh][])
 * [#7686](https://github.com/rubocop-hq/rubocop/pull/7686): Add new `JUnitFormatter` formatter based on `rubocop-junit-formatter` gem. ([@koic][])
 * [#7715](https://github.com/rubocop-hq/rubocop/pull/7715): Add `Steepfile` to default `Include` list. ([@ybiquitous][])
+* [#7700](https://github.com/rubocop-hq/rubocop/pull/7700): Add `space_after_colon` style to Layout/SpaceBeforeComma and Layout/SpaceInsideParens cops. ([@amilligan][])
```

Comment:
It is preferable that cop names be surrounded by backticks.

```suggestion
* [#7700](https://github.com/rubocop-hq/rubocop/pull/7700): Add `space_after_colon` style to `Layout/SpaceBeforeComma` and `Layout/SpaceInsideParens` cops. ([@amilligan][])
```

Reply:
I've fixed up the original commit; this should be done.